docs/content/docs.md

```diff
@@ -410,14 +410,20 @@ downloading and re-uploading.
 Remotes which don't support server-side copy **will** download and
 re-upload in this case.
 
-Server side copies are used with `sync` and `copy` and will be
+Server-side copies are used with `sync` and `copy` and will be
 identified in the log when using the `-v` flag.  The `move` command
 may also use them if remote doesn't support server-side move directly.
 This is done by issuing a server-side copy then a delete which is much
 quicker than a download and re-upload.
 
-Server side copies will only be attempted if the remote names are the
-same.
+Server-side copies will only be attempted if the remote names are the
```

Comment:
This is actually up to the backend whether it allows cross config server side copies. For example Google Drive does if you pass the `--drive-server-side-across-configs`, same with `--onedrive-server-side-across-configs`.

Perhaps write

> Server-side copies may not be attempted if the remote names are different. This will ...

> However on most backends this  will download and re-upload files from A to B

?